src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php

```diff
@@ -107,15 +113,18 @@ final class GetOrderForViewingHandler implements GetOrderForViewingHandlerInterf
      * @param ImageTagSourceParserInterface $imageTagSourceParser
      * @param TranslatorInterface $translator
      * @param Context $context
+     * @param Locale $locale
      */
     public function __construct(
         ImageTagSourceParserInterface $imageTagSourceParser,
         TranslatorInterface $translator,
-        Context $context
+        Context $context,
```

Comment:
Since context was only used to get `$this->context->language->id` I think you can remove it

Reply:
Fixed